### PR TITLE
[POL-84] Redo library common

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

File: src/pages/library/common.js [hunk 1]
```diff
@@ -20,136 +21,80 @@ import { RequestDatasetAccessModal } from 'src/pages/library/RequestDatasetAcces
...
+    }
   },
   pill: {
     width: '4.5rem', padding: '0.25rem', fontWeight: 500, textAlign: 'center',
     border: '1px solid', borderColor: colors.dark(0.25), borderRadius: '1rem',
     backgroundColor: 'white'
   },
   pillHighlight: {
-    color: 'white', backgroundColor: colors.primary(1), borderColor: colors.primary(1)
+    color: 'white', backgroundColor: colors.accent(), borderColor: colors.accent()
```
Comment (zarsky-broad) [inline src/pages/library/common.js:55]: Primary wasn't passing contrast check for accessibility.

File: src/pages/library/common.js [hunk 2]
```diff
@@ -478,7 +331,7 @@ const makeCard = variant => workspace => {
         width: 87,
         ...Utils.cond(
           [name.toLowerCase().includes('covid'), () => ({ backgroundImage: `url(${covidBg})` })],
-          [variant === 'gatk', () => ({ backgroundColor: '#333', backgroundImage: `url(${gatkLogo})`, backgroundSize: undefined })],
+          [namespace === 'help-gatk', () => ({ backgroundColor: '#333', backgroundImage: `url(${gatkLogo})`, backgroundSize: undefined })],
```
Comment (zarsky-broad) [inline src/pages/library/common.js]: This namespace is used in all environments for official gatk workspaces.

File: src/pages/library/common.js [hunk 3]
```diff
@@ -20,136 +21,73 @@ import { RequestDatasetAccessModal } from 'src/pages/library/RequestDatasetAcces
...
-      title,
-      div({ style: { flexGrow: 1 } }),
-      icon(isOpened ? 'angle-up' : 'angle-down', { size: 18, style: { flex: 'none' } })
-    ]),
-    div({
-      style: { flex: 'none' }
-    }, [h(RCollapse, { isOpened }, [children])])
-  ])
+    backgroundColor: 'white',
+    ...(highlight ? { color: 'white', backgroundColor: colors.accent(), borderColor: colors.accent() } : {})
```
Comment (zarsky-broad) [inline src/pages/library/common.js:53]: Primary wasn't passing contrast check for accessibility.

**Review by kyuksel — APPROVED** [review]
Mob reviewed  🚀

**Review by dmohs — APPROVED** [review]
Beautiful refactor. Thank you.

**Pull request merged**